### PR TITLE
Some important changes

### PR DESCRIPTION
**Pull request by YagyikPrajapat**
One of the changes made by me is converting latin into engllish and other changes are Spelling changes which is quite important to be done!

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -176,10 +176,9 @@ <h3>Contact us</h3>
               </div>
               <div class="full white_fonts">
                 <p>
-                  Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed
-                  do eiusmod tempor incididunt ut labore et dolore magna aliqua.
-                  Ut enim ad minim veniam, quis nostrud exercitation ullamco
-                  laboris nisi ut aliquip
+                 Education is the process of facilitating learning, or the acquisition of knowledge, skills, values, beliefs, and habits.
+                             Educational methods include teaching, training, storytelling, discussion and directed research
```
Comment (und3fined-v01d) [inline contact.html:180]: Check formatting

Comment (und3fined-v01d) [reply contact.html:180]: It is a good practice to be more verbose with the commit subject. Reviewers should be able to understand what code will be about without reading the PR description by just reading the commit subject.

Comment (YagyikPrajapat) [reply contact.html:180]: Yeah definitely!! I will take care of it next time!

**Comment by rahul799** [comment]
@YagyikPrajapat you won't need to close it. If you want to do the changes. You would have added add more commits to this PR.
So take care next time :smile:

**Comment by YagyikPrajapat** [comment]
Ok 😁
Thanks for letting me know
I am new to this so guide me!!

**Pull request closed without merging**